app/views/notifications/_emergency_banner.html.erb

```diff
@@ -1,7 +1,9 @@
 <div class="govuk-emergency-banner <%= banner.campaign_class %>" role="banner">
   <div>
     <h2><%= banner.heading %></h2>
-    <p><%= banner.short_description %></p>
+    <% unless banner.short_description.blank? %>
```

Comment:
Would `if banner.short_description` be better? Also consistent with the check for `banner.link`.